### ISSUE TITLE
Can't edit input fields in Ableton Live

### ISSUE THREAD

**Issue by snugsound** [Iunusov/ShoutVST#1]
**Steps to reproduce**
- add an instance of the VST to an audio channel in Ableton Live 9.71 (64-bit)
- open the plug-in editor
- attempt to edit any of the input fields

Note that I can click into the fields, and a cursor is shown, but they won't respond to keystrokes. Also note that radio buttons and buttons _do_ work.

Somebody else on KVR reported this issue as well, FYI: https://www.kvraudio.com/product/shoutvst-by-r-tur

**Comment by Iunusov** [comment]
yeah, that issue is related to GUI framework (FLTK). I will check.

**Comment by Bombe** [comment]
The most reliable way I found to access fields fast is to click into the Metadata field and then tab/shift-tab my way around the dialog…

**Comment by juilijoel** [comment]
I have the same problem
